### ISSUE TITLE
Models and Tables

### ISSUE THREAD

**Issue by njemanuel01** [njemanuel01/06-23-Finance-Manager#21]
# Tables/Models
### users(one to many with accounts_users, many to many with accounts)/User

```
| id(Int. Primary Key) | name(String) | total balance(Real) |
|----------------------|--------------|---------------------|
|1                     |Nick          |1234.56              |
```
### accounts_users(many to one with users, many to one with accounts)

```
| user_id(Int.) | account_id(Int.) |
|---------------|------------------|
|1              |2                 |
```
### accounts(one to many with accounts_users, many to many with users)/Account

```
| id(Int. Primary Key) | name(Text) | balance(Real) |
|----------------------|------------|---------------|
|1                     |Checking    |12.34          |
```
### categories(one to many with transactions)/Category

```
| id(Int. Primary Key) | name(Text) |
|----------------------|------------|
|1                     |Groceries   |
```
### transactions(many to one with accounts, many to one with categories, many to one with users(via accounts_users))/Transaction

```
| id(Int Primary Key) | amount(Real) | description(Text) | date(Text) | category_id(Int. Foreign Key) | account_id(Int. Foreign Key) |
|---------------------|--------------|-------------------|------------|-------------------------------|------------------------------|
|1                    |100           |Deposit from work  |06/23/15    |1                              |1                             | 
```

**Comment by sumeetjain** [comment]
Looks pretty close, but please check the rules (https://github.com/omahacodeschool/wiki/blob/master/ruby/naming-things.md) – I think a few things are not specified correctly.

Also, mind using some additional Markdown to make some things bold or otherwise standout? It's a bit hard to follow. https://help.github.com/articles/github-flavored-markdown/#tables might be helpful.
